compiler.js

```diff
@@ -339,6 +340,10 @@
                 else if (match.dec) {
                     section = this.addAttribute(match.dec[1] + '-=' + (match.dec[2] === undefined ? '1' : match.dec[2]), story, section, passage, isFirst, inputFilename, lineCount);
                 }
+                else if (match.deactivate) {
+                    section = this.addJS('$("[data-passage|=' + match.deactivate[1] + ']").addClass("disabled"); $("[data-passage|=' + match.deactivate[1] + ']").attr("tabindex", -1); squiffy.set("_turncount", squiffy.get("_turncount") + 1)', 
```

Comment:
Okay, that's fair enough. I haven't looked at this since that Pull request 28 days ago but I will try and refresh my mind of how all this works and see what I can do.